submissions/andrewklmn/html-css-popup/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="app.css" rel="stylesheet"/>
+  <title>html-css-popup-task</title>
+</head>
+<body>
+  <nav>
+    <div class="topnav-right">
+      <a href="#">Mail</a>
+      <a href="#">Images</a>
```

Comment:
Navigation is best used in this format, the sample from [MDN](https://developer.mozilla.org/uk/docs/Web/HTML/%D0%95%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82/nav)
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```